apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -114,14 +127,18 @@ const TokenCheckbox: React.FunctionComponent = () => {
   return (
     <View>
       <HoverCheckbox label="A checkbox with checkmark visible on hover" onChange={onChangeUncontrolled} />
-      <CircleColorCheckbox label="A circular token-customized checkbox" shape="circular" onChange={onChangeUncontrolled} defaultChecked />
+      {isAndroid ? ( // Circular checkbox not available on Android
+        <></>
+      ) : (
+        <CircleColorCheckbox label="A circular token-customized checkbox" shape="circular" onChange={onChangeUncontrolled} defaultChecked />
+      )}
       <BigLabelCheckbox label="A checkbox with a bold large font label" />
       <ComposedCheckbox label="A checkbox with a hot pink label and no padding" />
 
       <BlueCheckbox
         label="Token-customized checkbox. Customizable below."
         onChange={onChangeUncontrolled}
-        labelPosition="before"
+        labelPosition={isAndroid ? 'after' : 'before'}
```

Comment:
Why this change?

Reply:
We're not going to support the label position before the checkbox. So just reused existing example of Token Customisation.

Reply:
Reverted for now , till we have new way to handle differences.